examples/app-basic/tests/appNavigation.spec.ts

```diff
@@ -0,0 +1,31 @@
+import pluginJson from '../src/plugin.json';
+import { test, expect } from '@grafana/plugin-e2e';
+
+test.describe('navigating app', () => {
```

Comment:
I was thinking about wrapping this in some kind of page model as well but feels a bit overkill, wdyt @sunker?

Reply:
Yeah probably overkill if we only want to verify that they render successfully.